automatic/libreoffice-streams/README.still.md

```diff
@@ -7,3 +7,5 @@ LibreOffice is the free power-packed Open Source personal productivity suite for
 - This package installs LibreOffice Still which is the stable version of LibreOffice that has undergone more testing (over a longer time). This version is recommended for more conservative users or for deployments in enterprise or corporate environments. If you want the latest version with the most recent bleeding-edge features, install [libreoffice-fresh](/packages/libreoffice-fresh) instead.
 - For business deployments, The Document Foundation (the foundation behind the LibreOffice project) strongly recommends [support from certified partners](https://www.libreoffice.org/get-help/professional-support/) which also offer long-term support versions of LibreOffice.
 - Older versions of this package are likely broken because the LibreOffice team removes the download links for older versions after each update.
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to. **
```

Comment:
```suggestion
**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Otherwise it won't show up correctly as bold text.